components/listings/image-carousel.tsx

```
@@ -0,0 +1,156 @@
+                size="icon"
+                className="h-8 w-8 rounded-full bg-background/80"
+                onClick={toggleMute}
+              >
+                {isMuted ? <VolumeX className="h-4 w-4" /> : <Volume2 className="h-4 w-4" />}
+              </Button>
+            </div>
+          </div>
+        ) : (
+          <img
+            src={images[activeIndex] || "/placeholder.svg?height=400&width=600"}
+            alt={`${title} - Image ${activeIndex + 1}`}
```

Biome (1.9.4)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)